amit/html/Assigment6/Assig6.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang ="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignmnt6</title>
+    <style>
```

Comment:
check indentation

Reply:
mam i delete this file by mistake i push on git Assig6.html and Assigmt61